pdf-aam/index.html

```diff
@@ -191,121 +141,567 @@ <h2>General rules</h2>
 
         <section id="mapping_role_table">
           <h3>Role Mapping Tables</h3>
-          <h4 id="role-map-example"><code>example</code></h4>
-          <table class="data" aria-labelledby="role-map-example">
-            <tbody>
-              <tr>
-                <th>ARIA Specification</th>
-                <td>
-                  <a class="role-reference" href="#example"><code>example</code></a>
-                </td>
-              </tr>
-              <tr>
-                <th>MSAA + IAccessible2</th>
-                <td>
-                  <span class="property">Role: <code>ROLE_SYSTEM_EXAMPLE</code></span
-                  ><br />
-                </td>
-              </tr>
-              <tr>
-                <th><abbr title="User Interface Automation">UIA</abbr></th>
-                <td>
-                  <span class="property">Control Type: <code>Example</code></span
-                  ><br />
-                </td>
-              </tr>
-              <tr>
-                <th><abbr title="Accessibility Toolkit">ATK</abbr>/<abbr title="Assistive Technology - Service Provider Interface">AT-SPI</abbr></th>
-                <td>
-                  <span class="property">Role: <code>ROLE_EXAMPLE</code></span
-                  ><br />
-                </td>
-              </tr>
-              <tr>
-                <th>
-                  <abbr title="macOS Accessibility Protocol">AX API</abbr><sup>[<a href="#ftn.note1">Note 1</a>]</sup>
-                </th>
-                <td>
-                  <span class="property">AXRole: <code>AXxample</code></span
-                  ><br />
-                  <span class="property">AXSubrole: <code></code></span><br />
-                </td>
-              </tr>
-            </tbody>
-          </table>
-
-          <div class="note" id="ftn.note1">
-            <p>
-              <sup>[Note 1]</sup>
-              Footnote content
-            </p>
-          </div>
-        </section>
-      </section>
+          <h4 id="role-document"><code> Document (Child of Structure tree root)</code></h4>
+			<table class="data" aria-labelledby="role-document">
+				<tbody>
+					<tr>
+						<th>ARIA Specification</th>
+						<td>
+							<a class="role-reference" href="https://www.w3.org/TR/core-aam-1.2/#role-map-generic"><code>generic </code></a>
+						</td>
+					</tr>
+					<tr>
+						<th>Analogous HTML Element</th>
+						<td>
+							<span class="property" href="https://www.w3.org/TR/html-aam-1.0/#el-body">Role: <code>body</code></span>
+						</td>
+					</tr>
+					<tr>
+						<th>MSAA + IAccessible2</th>
+						<td>
+							<div class="general">Use WAI-ARIA mapping</div>
```

Comment:
I'd prefer to avoid use of divs here. This may conflict with the aria-common scripts that we are using.

```suggestion
Use WAI-ARIA mapping
```